src/Setup/WixBA/RootView.xaml

```diff
@@ -1,125 +1,487 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
-
+<!--  Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.  -->
 
 <Window x:Class="WixToolset.UX.RootView"
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:ux="clr-namespace:WixToolset.UX"
-    WindowStartupLocation="CenterScreen"
-    WindowStyle="None"
-    AllowsTransparency="True"
-    Background="{x:Null}"
-    Width="400"
-    Height="400">
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:i="http://schemas.microsoft.com/expression/2010/interactivity"
+        xmlns:ux="clr-namespace:WixToolset.UX"
+        MinWidth="550"
+        MinHeight="400"
+        MaxWidth="1200"
```

Comment:
If the WebBrowser control is the issue, then let's put a max size on that instead of the window (or just make the WebBrowser a fixed size?).